data/data-pipeline/data_pipeline/tests/sources/us_army_fuds/data/output.csv

```diff
@@ -0,0 +1,16 @@
+GEOID10_TRACT,Count of eligible Formerly Used Defense Site (FUDS) properties centroids,Count of ineligible Formerly Used Defense Site (FUDS) properties centroids,Boolean expressing whether any eligible Formerly Used Defense Sites (FUDS) have centroids in the tract
+06027000800,3.0000000000,14.0000000000,True
```

Comment:
I just realized these should be ints, not floats, so I need to add a line somewhere.